app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,35 @@
+class AdminMailer < ActionMailer::Base
+  LOG_TAG = "AdminMailer"
+  default :from => "no-reply@mylibrarynyc.org"
+
+  # Sends an email to let admins know that a request to create/update a bib(s) failed
+  def failed_bibs_controller_api_request(request_body, error_code_and_message, method_name)
+    begin
+      @action_name = action_name
+      @request_body = request_body
+      @error_code_and_message = error_code_and_message
+      emails = AdminUser.pluck(:email)
+      Rails.logger.debug("#{LOG_TAG}.admin_notification: About to send failed_bibs_controller_api_request email")
```

Comment:
Add the Logger Module for this mailer?

Reply:
Good call.  I added `include LogWrapper` to all the mailers and replaced `Rails.logger...` with `LogWrapper.log`.